test/e2e/testdata/empty_storageclass/manifest.yaml

```
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: test-empty-storageclass
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 1Gi
+  local:
+    path: /dev/null
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - topolvm-e2e-worker
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: ""
+  volumeMode: Block
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-empty-storageclass
+  namespace: "%s"
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: ""
+  volumeMode: Block
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-empty-storageclass
+  namespace: "%s"
+  labels:
+    app.kubernetes.io/name: pause
+spec:
+  containers:
+    - name: ubuntu
+      image: ubuntu:22.04
+      command:
+        - bash
+        - -c
+        - |
+          sleep inf &
+          trap "kill -SIGTERM $!" SIGTERM
+          wait $!
+          exit
+      volumeDevices:
+        - name: my-volume
+          devicePath: /dev/testing
+  volumes:
+    - name: my-volume
+      persistentVolumeClaim:
+        claimName: test-empty-sc
```

Isn't it a mistake for "test-empty-storageclass"?
```suggestion
        claimName: test-empty-storageclass
```

---

Why CI passing...?

---

> Why CI passing...?

https://github.com/topolvm/topolvm/pull/946#discussion_r1716641781